app/views/view_space.erb

```diff
@@ -4,4 +4,17 @@
     <div>
         Available From: <%= @space.format_date(@space.date_from) %> Available To: <%= @space.format_date(@space.date_to) %> 
     </div>
+
+    <form action='/request/new' method='post'>
+        <label for='start_date'>
+            Start Date:
+            <input type='date' name='start_date' />
+        </label>
+
+        <label for='end_date'>
+            End Date:
+            <input type='date' name='end_date' />
+        </label>
+        <input type='hidden' name='space_id' value= <%= @space.id %> />
+        <input type='submit' value='Request a booking'>
```

Comment:
We didn't have the User table associated with the Spaces one because it wasn't necessary for the user story. We planned to do this in the next one when the users could see requests for each of their spaces. If the relationship had already been set up, we'd have done this but we didn't need to.